homeassistant/components/switcher_kis/switch.py

```diff
@@ -154,3 +174,89 @@ async def async_turn_on_with_timer_service(self, timer_minutes: int) -> None:
         await self._async_call_api(API_CONTROL_DEVICE, Command.ON, timer_minutes)
         self.control_result = True
         self.async_write_ha_state()
+
+
+class SwitchereShutterChildLockBaseSwitchEntity(SwitcherEntity, SwitchEntity):
+    """Representation of a Switcher shutter base switch entity."""
+
+    _attr_device_class = SwitchDeviceClass.SWITCH
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_icon = "mdi:lock-open"
+    _cover_id: int
+
+    def __init__(self, coordinator: SwitcherDataUpdateCoordinator) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self.control_result: bool | None = None
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """When device updates, clear control result that overrides state."""
+        self.control_result = None
+        self.async_write_ha_state()
```

Comment:
```suggestion
        super._handle_coordinator_update()
```

Reply:
I am not sure if this is related to this PR.
This is how it is in many places already like:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/switcher_kis/button.py#L142
https://github.com/home-assistant/core/blob/dev/homeassistant/components/switcher_kis/climate.py#L126

@thecode what do you think?

Reply:
It should be fixed in the other places as well (in a new PR)